next-netflix/app/Search/page.tsx

```diff
@@ -0,0 +1,189 @@
+    }
+  }, [data]);
+
+  useEffect(() => {
+    setIsDataLoading(isQueryLoading);
+  }, [isQueryLoading]);
+
+  return (
+    <div className={styles.container}>
+      <SearchBar>
+        <AiOutlineSearch style={{color : "#C4C4C4", width : '30px'}} />
+        <SearchInput 
+        placeholder="Search for a show, movie, genre, e.t.c" 
+        value={searchInput}
+        onChange={(e) => setSearchInput(e.target.value)}/>
+        <GrFormClose style={{color : "red", width : '30px'}} />
+      </SearchBar>
+      
+      <ListWrapper>
+        <TitleText>Top Searches</TitleText>
+        {!isQueryLoading && !isDataLoading && !isFetching && data && data.pages && (
+          <List>
+          {searchResults.map((movie) => {
+            console.log(movie); // movie 로그 출력
+            return movie.results.map((result) => {
+              console.log(result); // result 로그 출력
+              return (
+                <div key={result.id}>
+                  {result.id ? (
+                    <InnerList>
+                      <Img src={`https://image.tmdb.org/t/p/original${result.poster_path}`} alt={result.title} />
+                      <MovieTitle>{result.title}</MovieTitle>
+                      <AiOutlinePlayCircle size="24"/>
+                    </InnerList>
+                  ) : (
+                    <h3>No poster and title available</h3>
+                  )}<div ref={bottom} />
+                </div>
+              );
+            });
+          })}
+          <div ref={bottom} />
+        </List>
+        )}
+      </ListWrapper>
+      <Navigation/>
+    </div>
+  );
+}
+  
+
+const SearchBar = styled.div`
+  display: flex;
+  width: 375px;
+  height: 52px;
+  align-items: center;
+  justify-content: space-around;
+  left: 0px;
+  top: 44px;
+  background: #424242;
+  margin: 44px 0px 15px;
+`;
+
+const SearchInput = styled.input`
+  background: none;
+  font-size: 15.213px;
+  width: 270px;
+  outline: none;
+`;
+
+const ListWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+
+const List = styled.div`
+  width: 375px;
+  background: #424242;
+  display: flex;
+  flex-direction: column;
+`;
+
+const InnerList = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 3px;
+`;
+
+const Img = styled.img`
+  background-color: rgb(25, 25, 25);
+  box-sizing: content-box;
+  width: 146px;
+  height: 76px;
+  aspect-ratio: 146 / 76;
+  border: 0px;
+  border-radius: 2px;
+  object-fit: cover;
+}
+`;
+
+const TitleText = styled.div`
+  font-size: 27px;
+  color: white;
+  font-weight: 700;
+  margin-left: 16px;
+  margin-bottom: 14px;
+`
+
+const MovieTitle = styled.div`
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14.7222px;
+  line-height: 30px;
+  letter-spacing: 0.751111px;
+  padding-left: 15px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  color: white;
+`
```

Comment:
MovieTitle이 길어지면 
![image](https://github.com/CEOS-Developers/next-netflix-17th/assets/15906101/08b2ff5b-4fb6-4187-9c37-0a4b81d34296)
옆 Play 버튼이 작아질 수 있어서 MovieTitle이 정해진 영역만 차지하도록 width를 주는 건 어떨까요 ?
```suggestion
const MovieTitle = styled.div`
  font-style: normal;
  font-weight: 400;
  font-size: 14.7222px;
  line-height: 30px;
  letter-spacing: 0.751111px;
  padding-left: 15px;
  overflow: hidden;
  text-overflow: ellipsis;
  white-space: nowrap;
  color: white;
  width : 180px;
`
```

Reply:
헛 제가 무한스크롤 기능 구현에 초점을 맞추느라 디자인 부분을 세세하게 체크하지 못했네요;-; 코드에 반영하도록 하겠습니다! 감사합니다🙌